src/test/resources/application-junit.yml

```diff
@@ -53,13 +53,16 @@ integration:
         - channelA
         - channelB
         - channelC
+scheduler:
+  opene:
+    cron:
+      expression: '0 * * * * *'
```

Comment:
Should the junit profile execute the scheduled job every minute, or can it be tured of (not sure if the scheduled job might interfere with for example db tests if it triggers at the same time as test is executing)?

Reply:
It shoiuld be turned off.